Gemfile

```diff
@@ -0,0 +1,43 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.2'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.1.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```

Comment:
In order to have a client–server database , you can use PostgreSQL instead of sqlite3
Gemfile --> gem 'pg'
/config/database.yml --> set as default PostgreSQL for dev, prod and test enviroment